src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -399,6 +401,15 @@ internal void ApplyReplacementText(string replacementText, bool propagateEditImm
                 }
 
                 _isApplyingEdit = false;
+
+                // Kick off the conflict resolution task after we are certain the replacement text
+                // has been propagated to all of the open buffers. Otherwise, the propagation can
+                // cause source control checkout dialogs, which pump and then run the application
+                // of the conflict resolution results while we are still applying the replacement
+                // text, which can cause crashes.
+                // See https://devdiv.visualstudio.com/DevDiv/_workitems?_a=edit&id=227513
+
+                UpdateConflictResolutionTask();
```

Comment:
Does this actually _start_ the conflict resolution stuff in the background (which we could have done in parallel with the application of text above) or does this just re-chain the update to the UI?